components/image/index.tsx

```diff
@@ -64,6 +64,7 @@ const Image: CompositionImage<ImageProps> = (props) => {
       ),
       icons,
       ...restPreviewProps,
+      rootClassName: rootClassName ? classNames(rootClassName, hashId, cssVarCls, rootCls) : mergedRootClassName,
```

Comment:
```suggestion
      rootClassName: classNames(rootClassName, hashId, cssVarCls, rootCls),
```

那这样就行了，不要使用上面的 mergedRootClassName。

Reply:
那确实是。

Reply:
```suggestion
```suggestion
      rootClassName: classNames(mergedRootClassName, rootClassName),
```

这样应该对了。
```

Reply:
👌